test_files/OSZICAR_MD

```diff
@@ -0,0 +1,45 @@
+       N       E                     dE             d eps       ncg     rms          rms(c)
```

Comment:
Do we actually need the new test file? Looks like `test_files/OSZICAR` also has a `mag` column.

Reply:
That's the beauty of this new parsing method! Actually we don't, but the old test doesn't consider other fields like `SP`, `SK` from MD simulation. I vote for removing one of these test files.

Reply:
>  I vote for removing one of these test files.

Sure, go right ahead. Fewer files is better.